R/pkg/R/functions.R

```diff
@@ -2055,20 +2058,10 @@ setMethod("countDistinct",
 
 #' @details
 #' \code{concat}: Concatenates multiple input columns together into a single column.
-#' If all inputs are binary, concat returns an output as binary. Otherwise, it returns as string.
+#' The function works with strings, binary and compatible array columns.
```

Comment:
btw, what does "compatible array columns" mean?

Reply:
@felixcheung 
I copied the sentence from scala doc. I guess it means "array columns that have data type compatible with string and binary", but I am not sure.

Reply:
I think it basically mean the arrays of elements are compatibile and castable for a wider type. (e.g. array<int> array<long> -> array<long>)